lib/scan.ts

```diff
@@ -136,3 +140,13 @@ export async function scan(options: Options): Promise<PluginResponse> {
     throw new Error(`Could not scan C/C++ project: ${error}`);
   }
 }
+
+function getExcludePatterns(projectRoot: string, dotSnykPath: string) {
+  try {
+    const dotSnyk: DotSnykConfig = parseDotSnyk(dotSnykPath);
+    return [dotSnykPath, ...parseUnmanagedExcludes(projectRoot, dotSnyk)];
+  } catch (err) {
+    // TODO(Tundra): Inform user that we failed to parse .snyk
+  }
```

Comment:
maybe we should add an empty row between 150 and 151

Reply:
like it 👍

Reply:
lets make it happen :)